apps/web/pages/event-types/[type].tsx

```diff
@@ -262,6 +264,26 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
 
   const [requirePayment, setRequirePayment] = useState(eventType.price > 0);
   const [advancedSettingsVisible, setAdvancedSettingsVisible] = useState(false);
+  const [recurringEventDefined, setRecurringEventDefined] = useState(
+    eventType.recurringEvent && eventType.recurringEvent.count !== undefined
+  );
+  const [recurringEventInterval, setRecurringEventInterval] = useState(
+    (eventType.recurringEvent && eventType.recurringEvent.interval) || 1
+  );
+  const [recurringEventFrequency, setRecurringEventFrequency] = useState(
+    (eventType.recurringEvent && eventType.recurringEvent.freq) || RRuleFrequency.WEEKLY
+  );
+  const [recurringEventCount, setRecurringEventCount] = useState(
+    (eventType.recurringEvent && eventType.recurringEvent.count) || 12
+  );
+
+  /* Just yearly-0, monthly-1 and weekly-2 */
+  const recurringEventFreqOptions = Object.entries(RRuleFrequency)
```

Comment:
Grabbing what we need from the frequency rrule enum, which as a good practice, they have `number to string` and `string to number` representations, so just keeping `number to string` and equal/below 3 to support the correct frequencies.